demo.html

```diff
@@ -5,22 +5,67 @@
         #canvas {
           border: 1px solid black;
         }
+        @keyframes spinner {
+          to {transform: rotate(360deg);}
+        }
+         
+        .spinner:before {
+          content: '';
+          box-sizing: border-box;
+          position: absolute;
+          top: 50%;
+          left: 50%;
+          width: 120px;
+          height: 120px;
+          margin-top: -10px;
+          margin-left: -10px;
+          border-radius: 50%;
+          border: 2px solid #ccc;
+          border-top-color: #333;
+          animation: spinner .6s linear infinite;
+        }
+
+        #blur_score {
+          font-size: 20px;
+          color: red;
+        }
       </style>
     </head>
     <body>
         <h1>Inspector Bokeh demo</h1>
         <form>
-            <label for="upload_image">Upload an image to measure its blurriness:</label>
-            <input type="file" id="upload_image">
+            <label for="upload_image">
+              Upload an image to measure its blurriness:
```

Comment:
We coud just go with "blur" here, blurriness doesn't seem to be in the dictionary.